npc/custom/woe_controller.txt

```diff
@@ -182,29 +182,30 @@ OnReward:
 			if (!.@gid) continue;
 
 			getguildmember( .@gid, 1, .@cid );
-			.@size = getguildmember( .@gid, 2, .@aid );
+			.@size_guild = getguildmember( .@gid, 2, .@aid );
 
 			if( .Options&4 ){
 				.@master_cid = getguildmasterid( .@gid );
-				
-				for( .@j = 0; .@j < .@size; .@j++ ){
-					if( .@cid[.@j] == .@master_cid ){
-						.@master_aid = .@aid[.@j];
+				.@index = inarray(.@cid, .@master_cid);
```

Comment:
```suggestion
				if((.@index = inarray(.@cid, .@master_cid)) != -1){
```

Reply:
also it need some <tab>s after that if you applied the changes automatically

Reply:
Like you say it shouldn't happen, I left the script without the index check on purpose to stop the script and debug just in case